### DISCUSSION TITLE
Welcome to BirdNetDisplay Discussions!

### DISCUSSION THREAD

**Discussion by ThomDyson** [ThomDyson/BirdNetDisplay#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BAM-Team** [comment]
That´s a very nice procect...excellent work :-D

**Comment by erikwiemann** [comment]
I like your project, would it be possible to convert it to use birdnet-go instead?

**Reply by ThomDyson** [reply]
Funny enough, I just ported it to BNG a few weeks ago. I wasn't planning on distributing it, but I guess I can. I don't use platformio anymore, so it is a simpler file set. Give me a few days to get a new repo setup.

**Reply by erikwiemann** [reply]
Sounds good. Thank you.

**Reply by ThomDyson** [reply]
The new repo is up https://github.com/ThomDyson/BirdNetGo-CYD

**Reply by erikwiemann** [reply]
Hi ThomDyson,

I downloaded and installed the code, it works perfectly. 

I had to adjust the lgvl conf file to make it fit my display.

Thank you very much.
